gym/hostname.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Why is this file necessary?

Reply:
ignore it, I was testing  slurm

Reply:
Ah, got it. If you are gonna further use/edit this then it's ok, otherwise I think you can delete already to not clutter things. up to you.